apps/meteor/ee/client/sidebar/footer/SidebarFooterWatermark.tsx

```diff
@@ -20,9 +23,13 @@ export const SidebarFooterWatermark = (): ReactElement | null => {
 				<Box fontScale='micro' color='hint' pbe={4}>
 					{t('Powered_by_RocketChat')}
 				</Box>
-				<Box fontScale='micro' color='pure-white' pbe={4}>
-					{t('Free_Edition')}
-				</Box>
+				{isLoading || isError ? (
+					<Skeleton width='5rem' height='1rem' />
```

Comment:
avoid `rem` as values to props, use numbers that represents pixels or percentage